locale/en/docs/guides/domain-postmortem.md

```diff
@@ -383,15 +383,15 @@ DataStream.prototype.data = function data(chunk) {
   // This code is self contained, but pretend it's a complex
   // operation that crosses at least one other module. So
   // passing along "this", etc., is not easy.
-  this.domain.run(function() {
+  this.domain.run(() => {
```

Comment:
Not necessarily part of this PR, but `domain` is deprecated. We should probably remove this.